packages/docusaurus-theme-classic/src/index.js

```diff
@@ -128,6 +128,7 @@ const swizzleAllowedComponents = [
   'SearchBar',
   'hooks/useTheme',
   'prism-include-languages',
+  'SearchPage',
```

Comment:
add it to search plugin itself.
```js
module.exports.getSwizzleComponentList = () =>['SearchPage']
```

Reply:
what do you mean?
where should i add this?
this file?
![image](https://user-images.githubusercontent.com/24778140/90235550-c270e400-de36-11ea-946c-8ac6d726d155.png)

Reply:
in` docusaurus-theme-search-algolia/index.js`

Reply:
@anshulrgoyal for now I think we should just allow swizzle, not recommend it, as it's likely we change a bit the implementation of this page, so not adding this array is fine

Reply:
But yes it shuld not be added to this list in the classic theme anyway